pkgs/generate-secrets/default.nix

```diff
@@ -1,6 +1,9 @@
 { pkgs }: with pkgs;
 
+let
+rpcauth = pkgs.callPackage ./rpcauth { };
```

Comment:
I think you can replace this with:
```
  rpcauth = pkgs.writeScriptBin "rpcauth" (builtins.readFile ./rpcauth/rpcauth.py);
```
(mind the indentation) and remove `rpcauth/default.nix`

Reply:
Fixed in fb94d19